### DISCUSSION TITLE
策略不挣钱？先来跑下回测试试？

### DISCUSSION THREAD

**Discussion by giscus** [zizhuspot/caoyuan#6] [General]
# 策略不挣钱？先来跑下回测试试？

收集历史行情数据,基于定义好的交易策略及参数,模拟在历史行情下执行策略对应的买卖点,计算交易盈亏情形。回测可以评估策略的实际效果,优化参数设定,选择有效的交易策略。良好的回测流程需要充分的行情数据、灵活的策略表达、严密的交易模拟、多维的指标分析。回测是确保策略有效性的基础,也是策略调优的主要手段。

https://www.quant1098.com/58a723e6ed14/

**Comment by youngjunning** [comment]
厉害，学废了
